interfaces/dotnet/examples/.template.config/template.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "http://json.schemastore.org/template",
+    "author": "Sammo Gabay",
```

Comment:
In other instances (e.g. License and website), Cantera does not attribute files to individual authors. Rather, it just mentions "Cantera Developers".

Reply:
Thanks for that. I myself am still learning myself how the "dotnet new templates" work. The goal is that the end user will be able to run `dotnet new ct-examples` and it will create a directory of runnable Cantera examples.

Reply:
I've changed the relevant line.

Reply:
I think you need to push …